doofinder-for-woocommerce/includes/class-migration.php

```diff
@@ -266,8 +266,14 @@ public static function transform_additional_attributes($additional_attributes)
         return $transformed_attributes;
     }
 
+    public static function create_token_auth()
+    {
+        $reset_credentials_context = new Reset_Credentials_Index();
+        $reset_credentials_context->reset_credentials();
```

Comment:
```suggestion
        $reset_credentials_context->reset_credentials();
```

no debería llamar a esta función?

Reply:
Tal cual, creo que lo cambié por un momento para hacer una prueba con la otra función y se me olvidó volverlo a poner, bien visto!